### PR TITLE
Readme mini update

### PR DESCRIPTION
**Pull request by vogella**

### CODE REVIEW

**Comment by azoitl** [comment]
> @azoitl do you see this? For some reason I cannot add you as reviewer which I wanted to do in this PR just for testing.

only after you directly mentioned me.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 # Eclipse Graphical Editing Framework (GEF)
 
-This repository contains the code base of the [Eclipse Graphical Editing Framework (GEF)](http://www.eclipse.org/gef/) project's legacy components [Draw2d 3.x](https://www.eclipse.org/gef/draw2d/index.php), [GEF (MVC) 3.x](https://www.eclipse.org/gef/gef_mvc/index.php), and [Zest 1.x](https://www.eclipse.org/gef/zest/index.php), which are still maintained but will not be developed further. The code base of the current GEF production components is located in the [eclipse/gef](https://github.com/eclipse/gef) repository instead.
+This repository contains the code base of the [Eclipse Graphical Editing Framework (GEF)](http://www.eclipse.org/gef/) project's classiic components [Draw2d 3.x](https://www.eclipse.org/gef/draw2d/index.php), [GEF (MVC) 3.x](https://www.eclipse.org/gef/gef_mvc/index.php), and [Zest 1.x](https://www.eclipse.org/gef/zest/index.php).
```
Comment (azoitl) [inline README.md:3]: there is a small typo in classic. Can I directyl edit that here?

Comment (vogella) [reply README.md:3]: Not sure if that work. Try or send another PR

**Comment by vogella** [comment]
Please try to subscribe to the repo notifications.

File: README.md (hunk 1 above)
Comment (CapXilinx) [reply README.md:3]: It's only possible to give permission to do this to the receiving repositories maintainer, what I read here https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Comment (azoitl) [reply README.md:3]: I googled a bit and it looks like the easiest is if I make a new pull request. But as I would like to update a few more things in the readme it is best if you merge and I work onwards from that.

Comment (vogella) [reply README.md:3]: :-)

**Pull request merged**